src/modules/new-request-form/translations/en-us.yml

```
@@ -0,0 +1,134 @@
+      screenshot: "https://drive.google.com/file/d/1T_2Eahrfs8iITZWvgAR6wxrxEdsy2D2G/view?usp=drive_link"
+      value: "Choose a file or drag and drop here"
+  - translation:
+      key: "new-request-form.attachments.stop-upload"
+      title: "Tooltip for the button for stopping the upload of an attachment"
+      screenshot: "https://drive.google.com/file/d/1Y6yeqp3fbfJbuq7tT8VutsLa80GZju-V/view?usp=drive_link"
+      value: "Stop upload"
+  - translation:
+      key: "new-request-form.attachments.remove-file"
+      title: "Tooltip for the button for removing an attachment"
+      screenshot: "https://drive.google.com/file/d/1JFVE2twzUkg-U6gGAs2EHqD-3jPvtJDA/view?usp=drive_link"
+      value: "Remove file"
+  - translation:
+      key: "new-request-form.attachments.uploading"
+      title: "[A11Y] Hidden label for screen readers for the upload attachment progress bar"
+      screenshot: "https://drive.google.com/file/d/1Y6yeqp3fbfJbuq7tT8VutsLa80GZju-V/view?usp=drive_link"
+      value: "Uploading {{fileName}}"
+  - translation:
+      key: "new-request-form.submit"
+      title: "Submit button for the New Request Form"
+      screenshot: "https://drive.google.com/file/d/1VuVYpxGA-uAGbvscl2WXTJ24yK5QydhZ/view?usp=drive_link"
+      value: "Submit"
+  - translation:
+      key: "new-request-form.cc-field.email-removed"
+      title: "[A11Y] Announcement read by screen readers when an e-mail is removed from the CC field."
+      screenshot: "https://drive.google.com/file/d/19iWawEvm6PJw1D2M7I1b32ntMDb7GqNw/view?usp=drive_link"
+      value: "{{email}} has been removed"
+  - translation:
+      key: "new-request-form.cc-field.emails-added"
+      title: "[A11Y] Announcement read by screen readers when an e-mail is added to the CC field."
+      screenshot: "https://drive.google.com/file/d/19iWawEvm6PJw1D2M7I1b32ntMDb7GqNw/view?usp=drive_link"
+      value: "{{emails}} have been added"
+  - translation:
+      key: "new-request-form.cc-field.email-added"
+      title: "[A11Y] Announcement read by screen readers when multiple e-mails are added to the CC field. The emails variable contains a comma-separated list of e-mails"
+      screenshot: "https://drive.google.com/file/d/19iWawEvm6PJw1D2M7I1b32ntMDb7GqNw/view?usp=drive_link"
+      value: "{{emails}} have been added"
+  - translation:
+      key: "new-request-form.cc-field.email-label"
+      title: "[A11Y] Hidden label for screen readers for an e-mail tag present in the CC field."
+      screenshot: "https://drive.google.com/file/d/19iWawEvm6PJw1D2M7I1b32ntMDb7GqNw/view?usp=drive_link"
+      value: "{{email}} - Press Backspace to remove"
+  - translation:
+      key: "new-request-form.cc-field.container-label"
+      title: "[A11Y] Hidden label for screen readers for the e-mails container in the CC field."
+      screenshot: "https://drive.google.com/file/d/19iWawEvm6PJw1D2M7I1b32ntMDb7GqNw/view?usp=drive_link"
+      value: "Selected CC e-mails"
```

```suggestion
      value: "Selected CC emails"
```
No hyphen needed: https://garden.zendesk.com/content/word-list#email-emails